generators/ci-cd/templates/_.gitlab-ci.yml

```diff
@@ -1,21 +1,19 @@
-<%_ if (integrations.includes('docker')) { _%>
```

Comment:
With default config in gitlab.com java image is needed (yes, Docker is used there).

Problems are:
* I must know that in gitlab.com by default is Docker used and I must check this
* in Jenkins I don't want Docker based config

Seems that separate choices are needed for Docker, maybe:
* Use Docker in Jenkins pipeline
* Use Docker in Gitlab CI (gitlab.com by default uses Docker)